newIDE/app/src/InAppTutorial/InAppTutorialTooltipDisplayer.js

```diff
@@ -369,7 +363,7 @@ const InAppTutorialTooltipDisplayer = ({
         },
       }}
       style={{
-        zIndex: getDisplayZIndexForHighlighter(anchorElement),
+        zIndex: aboveMaterialUiMaxZIndex,
```

Comment:
Tooltip used to be just above element, now as there's a blocking layer above everything, it needs to be at the same level as this layer, so it can be interacted with (to quit tutorial)

Reply:
I'm a bit afraid by this: in my memories, this helper is used to display something "just above" the element.
I remember some cases where we didn't want the highlighter to be displayed above everything else (if there is a dialog opened for instance, maybe the dialog opened when the user wants to quit the tutorial - I don't remember if there are other cases)

Reply:
I thought as well 🤔, but by redoing all of them + testing the flow to quit the tutorial, I didn't have any issue.
(now the tooltip is hidden when the dialog to quit appears, so maybe it's the reason why it works?)

Reply:
Tested again and you're right, if the blocking layer is not displayed, the tooltip becomes above everything.
I think I'll add a condition to display it above the element or above everything, depending on whether the blocking layer is displayed or not